src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.spec.ts

```diff
@@ -233,8 +238,13 @@ describe('InstructorHomePageComponent', () => {
 
     const courseButton: any = fixture.debugElement.nativeElement.querySelector('.btn-course');
     courseButton.click();
-    const archiveButton: any = fixture.debugElement.nativeElement.querySelector('.btn-delete-course');
-    archiveButton.click();
+    const deleteButton: any = document.querySelector('body > div > div > a.btn-delete-course.btn');
+      // Checking deleteButton
+      if (deleteButton) {
+        deleteButton.click();
+      } else {
+        console.error('Delete button not found');
+      }
```

Comment:
Let's remove debugging before merging in - just `deleteButton.click()` should be fine

Reply:
Sure I can do that